freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -96,15 +127,23 @@ def handle(self, event):
         # by the highest semantic version
         all_bundles = self._pyxis.get_latest_bundles(index_images)
 
-        bundles = self._pyxis.filter_bundles_by_related_image_digests(
-            original_images_digests, all_bundles)
-        bundle_digests = set()
-        for bundle in bundles:
-            if not bundle.get('bundle_path_digest'):
-                log.warning("Bundle %s doesn't have 'bundle_path_digests' set",
-                            bundle['bundle_path'])
+        # get bundle digests for original images
+        bundle_digests_by_related_nvr = {}
+        for image_nvr, image_digest in original_digests_by_nvr.items():
+            bundles = self._pyxis.get_bundles_by_related_image_digest(
+                image_digest, all_bundles
+            )
+            if not bundles:
+                log.info(f"No latest bundle image found for {image_nvr}")
                 continue
-            bundle_digests.add(bundle['bundle_path_digest'])
+
+            for bundle in bundles:
+                bundle_digest = bundle.get('bundle_path_digest')
```

Comment:
If I recall correctly, there is a bug that causes `bundle_path_digest` not to be set in the data returned by Pyxis. So we should guard against this like the previous code. If you've confirmed that this is no longer the case, then access the key directly to indicate that it is always set (i.e. don't use `.get`).